Makefile

```diff
@@ -359,3 +359,49 @@ test-lam-e2e: pkg/goods/bins/local-artifact-mirror
 bin/installer:
 	@mkdir -p bin
 	go build -o bin/installer ./cmd/installer
+
+# make test-embed channel=Unstable app=slackernews
+.PHONY: test-embed
+test-emded: export OS=linux
+test-embed: export ARCH=amd64
```

Comment:
This means we can't use this in the dev env no?

Reply:
it won't affect, this exports them only in this 1 new make target

Reply:
I understand. I'm saying we won't be able to benefit from it. Which might be fine